package.json

```diff
@@ -78,6 +80,7 @@
   "pre-commit": [
     "lint",
     "validate",
-    "test"
+    "test",
+    "check-dependencies"
   ]
-}
+}
```

Comment:
Can you regenerate the PRs so that the final newline isn't removed? Lots of tools warn if its not there so it would be nice to keep it around.